libpod/runtime.go

```diff
@@ -254,6 +255,8 @@ type RuntimeConfig struct {
 	// SDNotify tells Libpod to allow containers to notify the host
 	// systemd of readiness using the SD_NOTIFY mechanism
 	SDNotify bool
+	// CgroupCheck verifies if the cgroup check for correct OCI runtime has been done.
+	CgroupCheck bool `toml:"cgroup_check,omitempty"`
```

Comment:
Do we need the bool here? It doesn't seem to do much.

Reply:
Oh, nevermind, I see what you're doing here...

I don't really like it, but I don't see another easy way, and we can just remove it once we're done.